src/main/java/com/jmb/springfactory/model/entity/Task.java

```diff
@@ -31,9 +28,9 @@
 
   private TaskStatusEnum status;
 
-  private LocalDate startDate;
+  private Date startDate;
 
-  private LocalDate finishDate;
+  private Date finishDate;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "finishDate" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)